docs/msbuild/customize-your-build.md

```diff
@@ -15,11 +15,21 @@ ms.workload:
   - "multiple"
 ---
 # Customize your build
+
+MSBuild projects that use the standard build process (importing `Microsoft.Common.props` and `Microsoft.Common.targets`) have several extensibility hooks that can be used to customize your build process.
+
+## Adding arguments to command-line MSBuild invocations for your project
+
+A `Directory.Build.rsp` file in your source directory will be applied to command-line builds of your project. For details, see [MSBuild Response Files](../msbuild/msbuild-response-files.md#directorybuildrsp).
```

Comment:
Doesn't the searching for rsp files match the searching for props files? Thought that was confirmed in slack.

Reply:
Yes, I didn't mean to imply otherwise. Does my update here clarify? I'm trying to avoid duplicating the linked explanation.